ScoutSuite/__main__.py

```diff
@@ -113,6 +113,8 @@ def run(provider,
     """
 
     loop = asyncio.get_event_loop()
+    if loop.is_closed():
```

Comment:
Why are you adding this?

Reply:
gah, this is unrelated.  This is so it will run in a Lambda.

Reply:
I removed this.  I'll try to put it in another PR later